plugins/vppv2/puntplugin/README.md

```diff
@@ -1,29 +1,177 @@
 # Punt plugin
 
-Punt plugin defines [model](../model/punt/punt.proto) and supports three scenarios:
+**Written for: v1.7-vpp18.10**
 
-1. **IP punt redirect:** packet matching one of the VPP IP addresses but otherwise would be dropped will be send to 
-the defined next hop IP address via TX interface. The rule can be enhanced and input traffic filtered with RX 
-interface - redirect applies only to traffic received via RX.
+The punt plugin provides several options for how to configure the VPP to allow a specific IP traffic to be punted to the host TCP/IP stack. The plugin supports punt to the host (either directly, or via Unix domain socket) and registration of IP punt redirect rules.
 
-2. **Punt to host:** packet matching one of the VPP IP address and also defined L3 protocol, L4 protocol and also port
-is punted to host
+- [Punt to host stack](#Punt to host stack)
```

Comment:
Anchor links are always defined lowercase and with `-` instead of spaces.

Reply:
`-` is important (I forgot it somehow) but links do not need to be lowercased

Reply:
https://github.com/ligato/vpp-agent/blob/e5bc5d9c59b855e35602a574b85b85310f2604f3/plugins/vppv2/puntplugin/README.md#punt-plugin